### PR TITLE
Fix network event calls.

### LINKED ISSUE #570: Bad arguments to add_event on develop
I am seeing this in the overnight tests on develop:

```
Dec  1 23:19:56 jenkins-201201-la4feega-sf-5 ERROR sf-queues-1606864794.1813653-000[13111] [Exception] Ignored error in sf-queues: add_event() missing 2 required positional arguments: 'duration' and 'message'
Traceback (most recent call last):
  File "/usr/local/lib/python3.6/dist-packages/shakenfist/daemons/queues.py", line 81, in handle
    instance_start(instance_uuid, task.network())
  File "/usr/local/lib/python3.6/dist-packages/shakenfist/daemons/queues.py", line 224, in instance_start
    n.create()
  File "/usr/local/lib/python3.6/dist-packages/shakenfist/net.py", line 324, in create
    db.add_event('network', self.uuid, 'deploy', 'enqueued')
TypeError: add_event() missing 2 required positional arguments: 'duration' and 'message'
```

### PR DESCRIPTION
**Pull request by mikalstill**
These were inconsistent and sometimes just wrong. Fixes #570.

### CODE REVIEW

**Comment by mikalstill** [comment]
CI failure seems unrelated.

File: shakenfist/net.py [hunk 1]
```diff
@@ -321,7 +321,7 @@ def create(self):
             self.update_dhcp()
         else:
             db.enqueue('networknode', DeployNetworkTask(self.uuid))
-            db.add_event('network', self.uuid, 'deploy', 'enqueued')
+            self.add_event('network', self.uuid, 'deploy', 'enqueued')
```
Comment (mandoonandy) [inline shakenfist/net.py:324]: I think this correction would be better as `self.add_event('deploy', 'enqueued')`

**Review by mandoonandy — CHANGES_REQUESTED** [review]

File: shakenfist/net.py (hunk 1 above)
Comment (mikalstill) [reply shakenfist/net.py:324]: Oh yeah. Fair.

**Review by mandoonandy — APPROVED** [review]
LGTM

**Pull request merged**